### PR TITLE
Add "About this release" section for 5.11.0

### PR DESCRIPTION
**Pull request by NipuniBhagya**

### CODE REVIEW

File: en/docs/get-started/about-this-release.md [hunk 1]
```diff
@@ -2,18 +2,16 @@
...
-    <li><b><a href="../../learn/passwordless-authentication-using-fido2">Passwordless authentication support</a></b></li>
-    <li><b><a href="../../learn/user-portal">An improved User Portal</a></b></li>
-    <li><b><a href="../../develop/application-rest-api">New RESTful APIs for user self-services and server management</a></b></li>
-    <li><b><a href="../../develop/scope-based-authorization-for-internal-rest-apis">Scope based authorization for internal REST APIs</a></b></li>
-    <li><b>Introduce a unique user identifier across the system
-</b><br>
-Previously, the username of the user was considered as an immutable attribute across the system which had multiple limitations. Therefore a new unique user identifier has been introduced along with a new set of user core APIs.
-Also new user store managers have been introduced for all the types such as JDBC, Read-Only LDAP, Read-Write LDAP, and Active directory which can be compatible with the new unique user identifier implementation. 
-</li>
+    <li><b>New react based console application(BETA) with developer and administrator views to manage and maintain the features offered by the Identity Server.</b></li>
```
Comment (jeradrutnam) [inline en/docs/get-started/about-this-release.md:8]:
```suggestion
    <li><b>New react based console application (BETA) with developer and administrator views to manage and maintain the features offered by the Identity Server.</b></li>
```

**Review by jeradrutnam — APPROVED** [review]

File: en/docs/get-started/about-this-release.md [hunk 2]
```diff
@@ -2,18 +2,16 @@
...
+    <li><b>Enhanced My Account application for users to manage their account-related preferences.</b></li>
```
Comment (jeradrutnam) [inline en/docs/get-started/about-this-release.md:9]: Let say in 5.10.0 it was named as `User Portal`

Comment (sherenewso2) [reply en/docs/get-started/about-this-release.md:9]: and lets link it to `../../learn/my-account`

File: en/docs/get-started/about-this-release.md [hunk 3]
```diff
@@ -2,18 +2,16 @@
...
-    <li><b><a href="../../develop/application-rest-api">New RESTful APIs for user self-services and server management</a></b></li>
-    <li><b><a href="../../develop/scope-based-authorization-for-internal-rest-apis">Scope based authorization for internal REST APIs</a></b></li>
-    <li><b>Introduce a unique user identifier across the system
-</b><br>
-Previously, the username of the user was considered as an immutable attribute across the system which had multiple limitations. Therefore a new unique user identifier has been introduced along with a new set of user core APIs.
-Also new user store managers have been introduced for all the types such as JDBC, Read-Only LDAP, Read-Write LDAP, and Active directory which can be compatible with the new unique user identifier implementation. 
-</li>
+    <li><b>New react based console application (BETA) with developer and administrator views to manage and maintain the features offered by the Identity Server.</b></li>
+    <li><b>Enhanced My Account application for users to manage their account-related preferences.</b></li>
+    <li><b>Data protection with symmetric key encryption.</b></li>
```
Comment (sherenewso2) [inline en/docs/get-started/about-this-release.md:10]: let's link to `../../administer/symmetric-overview`

File: en/docs/get-started/about-this-release.md [hunk 4]
```diff
@@ -2,18 +2,16 @@
...
+    <li><b>Role management API.</b></li>
+    <li><b>CORS management APIs.</b></li>
```
Comment (sherenewso2) [inline en/docs/get-started/about-this-release.md:12]: link --> `../../develop/cors-rest-api`

**Pull request merged**